### DISCUSSION TITLE
Sayın yöneticiler orda mısınız?

### DISCUSSION THREAD

**Discussion by Zeyniaktass** [IBAX-io/go-ibax#1841] [Soru Cevap - Türkçe]
Sayın ibax yöneticileri lütfen bu konuya açıklık getirir misiniz, herkes neler olup bittiğinin farkında değil.  TG adminlerinden biri mevcut sıralama listesinin güncelleneceğini, diğeri ise güncellenmeyeceğini söylüyor.

 • Sorum şu, son 3 retweet puanı sayılır mı, sayılmaz mı?

 • Son 3 retweet olayı kasıtlı mıydı?  Cevabınız evet ise neden önceden haber vermediniz, niyetiniz neydi?

 • Tweetin altına 3 arkadaşını etiketle ve 200 puan al (her seferinde) bu puanlar verilecek mi?

 • Son retweet hatası nedeniyle 2 gün önce kayıt olan kişiler 2 aydır uğraşanların önüne geçti, bu mağduriyeti çözecek misiniz?

**Comment by yolandadadada** [comment]
Due to twitter restrictions, these extra points are limited on a first-come, first-served basis.

**Reply by Yusufknk** [reply]
İbax retweet görevleri için belirli sayıda API kontenjanı var , böylece ilk gelenler yapabilir. API dolduktan sonra Malasef diğer kişiler yapamayacak ve geçerli sıralamaya dahil olacaktır. Evet var olan sıralama güncellenecektir. Güncelleme daha sonraki etkinliklerle olacaktır.
